ext/standard/tests/strings/str_ends_with.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+str_ends_with() function - unit tests for str_ends_with()
+--FILE--
+<?php
+/* Prototype: bool str_ends_with (string $haystack, string $needle);
+   Description: Determine if $haystack ends with $needle
+*/
+$testStr = "beginningMiddleEnd";
+var_dump(str_ends_with($testStr, "End"));
+var_dump(str_ends_with($testStr, "end"));
+var_dump(str_ends_with($testStr, "en"));
+var_dump(str_ends_with($testStr, $testStr."a"));
+var_dump(str_ends_with($testStr, "a".$testStr));
```

Comment:
```suggestion
var_dump(str_ends_with($testStr, $testStr));
var_dump(str_ends_with($testStr, $testStr.$testStr));
```
(i.e. add a test that a string ends with itself, and merge both `$testStr."a"` and `"a".$testStr` into a single `$testStr.$testStr`)